exec/java-exec/src/main/resources/rest/generic.ftl

```diff
@@ -75,9 +78,6 @@
               <#if showLogs == true>
                   <li><a href="/logs">Logs</a></li>
               </#if>
-            </ul>
-            </#if>
-            <ul class="nav navbar-nav navbar-right">
```

Comment:
🤔 I see there are some variables here `showLogs, showStorage`
Is there any way to create a new variable `isShopify` or something like that, so you can add the new html elements like that?